### ISSUE TITLE
Segfault when ISTIO_QUIT_API fails to respond

### ISSUE THREAD

**Issue by jnicholls** [redboxllc/scuttle#19]
This is with Scuttle v1.2.1 (the latest as of this writing).

https://github.com/redboxllc/scuttle/blob/master/main.go#L135

```
scuttle: Stopping Istio using Istio API 'http://127.0.0.1:15020' (intended for Istio >v1.2)
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x7c2c07]

goroutine 1 [running]:
main.killIstioWithAPI()
	/home/runner/work/scuttle/scuttle/main.go:135 +0x287
main.kill(0x0)
	/home/runner/work/scuttle/scuttle/main.go:110 +0xcc
main.main()
	/home/runner/work/scuttle/scuttle/main.go:84 +0x31f
```

Attempting to dereference `resp` which is nil.

**Comment by linjmeyer** [comment]
Thanks for letting us know, fix is pretty easy: PR is #20

**Comment by jnicholls** [comment]
My pleasure, thanks for the quick fix.

**Comment by linjmeyer** [comment]
@jnicholls this is fixed in [v1.2.2](https://github.com/redboxllc/scuttle/releases/tag/v1.2.2)

**Comment by jnicholls** [comment]
@linjmeyer @jvasallo Thank you!
